src/App.tsx

```diff
@@ -15,37 +16,85 @@
   'Garlic',
 ];
 
+enum Way {
+  'Unknown' = 0,
+  'Info' = 'is-info',
+  'Success' = 'is-success',
+}
+
+function getSorted(obj: string[], way: Way) {
+  const currentGoods = [...obj];
+
+  currentGoods.sort((el1, el2) => {
+    switch (way) {
+      case 'is-info':
+        return el1.localeCompare(el2);
+      case 'is-success':
+        return el1.length - el2.length;
+      default:
+        return 0;
+    }
+  });
+
+  return currentGoods;
+}
+
 export const App: React.FC = () => {
+  const [selectItem, setSelectItem] = useState('');
```

Comment:
The `selectItem` state is being used as a `Way` type, but it is initialized as a string. Consider initializing it with a `Way` type or handling the type conversion more explicitly.